### PR TITLE
Added support for the new Invisible reCaptcha badge attribute and execute() API method.

### PR DESCRIPTION
**Pull request by bjornlaurell**
This should fix the following issue: #186

### CODE REVIEW

File: src/directive.js [hunk 1]
```diff
@@ -52,6 +53,7 @@
                         lang: scope.lang || attrs.lang || null,
                         tabindex: scope.tabindex || attrs.tabindex || null,
                         size: scope.size || attrs.size || null,
+						badge: scope.badge || attrs.badge || null,
```
Comment (MaximeCheramy) [inline src/directive.js]: Bad indent.

**Comment by mtrias** [comment]
Thank you for your contribution!

Is anyone else interested in this PR to be merged?

If yes, then I'd appreciate someone to test this, confirm that it works, and code review.

**Comment by MaximeCheramy** [comment]
@mtrias it looks good to me and I've been able to use the invisible recaptcha (not used in production).

File: src/service.js [hunk 2]
```diff
@@ -84,6 +84,14 @@
         provider.setLang = function(lang){
             config.lang = lang;
         };
+        /**
```
Comment (iambrosi) [inline src/service.js]: Nitpick: Can you add an empty line between provider methods?

File: src/service.js [hunk 3]
```diff
@@ -84,6 +84,14 @@
...
+         * Sets the reCaptcha badge position which will be used by default is not specified in a specific directive instance.
```
Comment (iambrosi) [inline src/service.js]: Nitpick 2:

```diff
-...by default is not specified...
+...by default if not specified...
```

File: src/service.js [hunk 4]
```diff
@@ -84,6 +84,14 @@
...
+         *
+         * @param lang  The reCaptcha badge position.
```
Comment (iambrosi) [inline src/service.js]: The parameter name is wrong.

**Comment by iambrosi** [comment]
Thanks @bjornlaurell !!

**Comment by iambrosi** [comment]
Tagged as [4.1.0](https://github.com/VividCortex/angular-recaptcha/releases/tag/4.1.0)

**Pull request merged**